src/formatDate.js

```diff
@@ -8,7 +8,28 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
```

Comment:
Consider making a defensive copy of the `fromFormat` and `toFormat` arrays before using `pop` to prevent side effects. This would ensure that the original arrays passed to the function are not modified, which could lead to unexpected behavior if those arrays are used elsewhere in the program after calling `formatDate`.